server-cloud/cmd/main.go

```diff
@@ -62,8 +67,20 @@ func main() {
 		logger.Fatal("error migrating database", zap.Error(err))
 	}
 
+	srvConf := server.Config{
+		Port:              conf.Port,
+		Auth0ClientID:     conf.Auth0ClientID,
+		Auth0ClientSecret: conf.Auth0ClientSecret,
+		Auth0CallbackURL:  conf.Auth0CallbackURL,
+		Auth0Domain:       conf.Auth0Domain,
+		SessionsSecret:    conf.SessionsSecret,
+	}
+
 	// Init server
-	server := server.New(logger, db)
+	server, err := server.New(logger, db, srvConf)
+	if err != nil {
+		logger.Fatal("error creating authenticator", zap.Error(err))
```

Comment:
Since this is returned from server, the message should be "error creating server"